crapssim/bet.py

```diff
@@ -275,31 +344,28 @@ class Place10(Place):
 
 
 class Field(WinningLosingNumbersBet):

```

Comment:
Field should be a one roll bet as well

Reply:
Good call, changed with [5806a70](https://github.com/skent259/crapssim/pull/20/commits/5806a7097273ede219f38227f1067b3016cbea58)